base/common/defines.h

```diff
@@ -1,5 +1,19 @@
 #pragma once
 
+/// __has_feature supported only by clang.
+///
+/// But libcxx/libcxxabi overrides it to 0,
+/// thus the checks for __has_feature will be wrong.
+///
+/// NOTE: that we cannot simply undefin __has_feature,
+/// since this will be broken if some C++ header will be included after
+/// including <common/defines.h>
+#if defined(__clang__) && defined(__has_feature)
+#    define ch_has_feature __has_feature
+#else
+#    define ch_has_feature(__x) 0
```

Comment:
But now we also define our macro to 0 on gcc, what is the difference?

Reply:
The "rebase error", thanks, fixed